test/e2e/step-definitions/generic-steps.ts

```diff
@@ -189,6 +189,23 @@ Then('I should see the {string} page', (pageTitle) => {
     .to.eventually.equal(pageTitle);
 });
 
+Given('I am on the {string} page', (pageTitle) => {
+  PageHelper.waitForOverlay('click-block-active');
+  // Wait for the page title to exist
+  PageHelper.getPageTitle(pageTitle);
+
+  // Check that it is the last page title i.e. the displayed one
+  return expect(PageHelper.getDisplayedPageTitle().getText(), `Expected displayedPageTitle to equal ${pageTitle}`)
+    .to.eventually.equal(pageTitle);
+});
+
+
```

Comment:
PLease remove all but one line of white space

Reply:
I have removed the space